pkl/project.go

```diff
@@ -46,8 +46,8 @@ type ProjectPackage struct {
 	Uri                 string   `pkl:"uri"`
 }
 
-// ProjectEvaluatorSettings is the Go representation of pkl.Project#EvaluatorSettings
-type ProjectEvaluatorSettings struct {
+// EvaluatorSettings is the Go representation of pkl.EvaluatorSettings
+type EvaluatorSettings struct {
```

Comment:
I don't feel great about the name clash between `EvaluatorOptions` and `EvaluatorSettings`.

Some other ideas:

* Move all of these to a sub-package; e.g. `stdlib` or something that.
* Name it `PklEvaluatorSettings`

I think let's keep this name for now, and we can handle this in another PR?